### PR TITLE
Better description of the binds parameter of connect

### PR DESCRIPTION
**Pull request by BenjaminNavarro**
Now states how binds are handled if they are used together with parameters passed to emit_signal

Sorry for the previous PR, I made a mistake with Git that closed it...

### CODE REVIEW

File: doc/classes/Object.xml [hunk 1]
```diff
@@ -139,7 +139,7 @@
 			<argument index="4" name="flags" type="int" default="0">
 			</argument>
 			<description>
-				Connects a [code]signal[/code] to a [code]method[/code] on a [code]target[/code] object. Pass optional [code]binds[/code] to the call as an [Array] of parameters. Use [code]flags[/code] to set deferred or one-shot connections. See [enum ConnectFlags] constants.
+				Connects a [code]signal[/code] to a [code]method[/code] on a [code]target[/code] object. Pass optional [code]binds[/code] to the call as an [Array] of parameters. These parameters will be passed after any parameter passed call to [method emit_signal]. Use [code]flags[/code] to set deferred or one-shot connections. See [enum ConnectFlags] constants.
```
Comment (akien-mga) [inline doc/classes/Object.xml]: I would rephrase to: "These parameters will be passed to the method after any parameter used in the call to [method emit_signal]."

Even then, it's not immediately obvious what this means, so I'd add an example of connecting a signal with binds + emitting it with a custom parameter, and showing what the signature of the callback should be like.

Comment (akien-mga) [reply doc/classes/Object.xml]: CC @NathanLovato

Comment (BenjaminNavarro) [reply doc/classes/Object.xml]: Yes you're right, it makes sense to provide an example. I'll make one and also rephrase as you proposed

**Comment by akien-mga** [comment]
I pushed a rebase to squash the two commits and fix the indentation. Should now be good to merge.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by BenjaminNavarro** [comment]
Glad I could contribute :smiley:
